components/Triggers/src/CompareStateTrigger.cs

```diff
@@ -13,7 +13,7 @@ namespace CommunityToolkit.WinUI;
     /// <para>
     /// Example: Trigger if a value is greater than 0
     /// <code lang="xaml">
-    ///     &lt;triggers:CompareStateTrigger Value="{Binding MyValue}" CompareTo="0" Comparison="GreaterThan" />
+    ///     &lt;triggers:CompareStateTrigger Value="{x:Bind MyValue}" CompareTo="0" Comparison="GreaterThan" />
```

Comment:
Don't know how this bit is used, but... Are you aware that the default mode of `x:Bind` is **OneTime**, in stead of OneWay for Binding?

Reply:
Good catch @Jay-o-Way, yes that's true. I just did a mindless replace here. This is just the doc comment example, but we should ensure its proper.